app/javascript/components/SummaryRow.js

```diff
@@ -35,6 +35,15 @@ export function SummaryRow(rebateForm, key, checked, checkIt) {
           }
         </td>
       }
+      {declined &&
+        <td className='rebate-results-table-cell' id='application-id'>
```

Comment:
I think the `id` attribute was accidentally copied here?

Reply:
nice catch :sparkles: